frontend/src/Components/Forms/LoginForm.test.js

```diff
@@ -0,0 +1,37 @@
+import { render } from '@testing-library/react'
+import user from '@testing-library/user-event'
+import LoginForm from './LoginForm'
+
+const onSubmitMock = jest.fn()
+
+describe('AddPlantForm', () => {
+  it('Calls onSubmit with correct data, resets form and calls onSubmitMock', () => {
```

Comment:
I would start here with lower case, because the sentence continues after the "it".